vagrant/k8s/Vagrantfile

```diff
@@ -107,6 +107,18 @@ SCRIPT
 cluster = read_cluster_config
 create_etc_hosts(cluster)
 
+def customize(v, id)
```

Comment:
Why is `id` passed in here?  It's not referenced in the body of this method anywhere